src/test/java/com/google/jenkins/plugins/storage/IT/StdoutUploadStepPipelineIT.java

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.jenkins.plugins.storage.IT;
+
+import static com.google.jenkins.plugins.storage.IT.ITUtil.deleteFromBucket;
+import static com.google.jenkins.plugins.storage.IT.ITUtil.dumpLog;
+import static com.google.jenkins.plugins.storage.IT.ITUtil.formatRandomName;
+import static com.google.jenkins.plugins.storage.IT.ITUtil.loadResource;
+import static org.junit.Assert.assertNotNull;
+
+import com.cloudbees.plugins.credentials.Credentials;
+import com.cloudbees.plugins.credentials.CredentialsStore;
+import com.cloudbees.plugins.credentials.SystemCredentialsProvider;
+import com.cloudbees.plugins.credentials.domains.Domain;
+import com.google.jenkins.plugins.credentials.oauth.GoogleRobotPrivateKeyCredentials;
+import com.google.jenkins.plugins.credentials.oauth.ServiceAccountConfig;
+import com.google.jenkins.plugins.storage.StringJsonServiceAccountConfig;
+import hudson.EnvVars;
+import hudson.model.Result;
+import hudson.slaves.EnvironmentVariablesNodeProperty;
+import java.util.logging.Logger;
+import org.jenkinsci.plugins.workflow.cps.CpsFlowDefinition;
+import org.jenkinsci.plugins.workflow.job.WorkflowJob;
+import org.jenkinsci.plugins.workflow.job.WorkflowRun;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.jvnet.hudson.test.JenkinsRule;
+
+public class StdoutUploadStepPipelineIT {
+  private static final Logger LOGGER = Logger.getLogger(StdoutUploadStepPipelineIT.class.getName());
+
+  @ClassRule public static JenkinsRule jenkinsRule = new JenkinsRule();
+  private static EnvVars envVars;
+  private static String projectId;
+  private static String credentialsId;
+  private static String bucket;
+  private static String pattern = "build_log.txt";
+
+  @BeforeClass
+  public static void init() throws Exception {
+    LOGGER.info("Initializing StdoutUploadStepPipelineIT");
+
+    projectId = System.getenv("GOOGLE_PROJECT_ID");
```

Comment:
I think a lot of this initialization is a good candidate for putting in ITUtil to reduce duplication.

Reply:
Done.